resources/js/features/achievements/components/CreateAchievementTicketMainRoot/CreateAchievementTicketMainRoot.test.tsx

```diff
@@ -807,82 +807,86 @@ describe('Component: CreateAchievementTicketMainRoot', () => {
     },
   );
 
-  it('sends along data from the ?extra query param if that data is provided', async () => {
-    // ARRANGE
-    const postSpy = vi.spyOn(axios, 'post').mockResolvedValueOnce({ data: { ticketId: 123 } });
+  it(
+    'sends along data from the ?extra query param if that data is provided',
+    { timeout: 10_000 },
```

Comment:
This line is the only actual change in this file. I was experiencing some flake locally because the test timed out. I increased the timeout duration and that seems to have resolved the issue.